ChaosMod/Effects/Effect.h

```diff
@@ -44,6 +44,11 @@ struct RegisteredEffect
 		return m_EffectIdentifier == effectIdentifier;
 	}
 
+	EffectIdentifier GetIndentifier()
```

Comment:
Return a const reference instead here as there's no reason to return a copy of EffectIdentifier, can also be marked as a const function.